.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: java
+sudo: false
+install: true
```

Comment:
Hi @bellingard , I've seen that the build failed. I don't see any relation to sonar.
Do you think the failure could be caused by the `sudo/install` configuration?